ownCloudScreenshotsTests/ownCloudScreenshotsTests.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  ownCloudScreenshotsTests.swift
+//  ownCloudScreenshotsTests
+//
+//  Created by Javier Gonzalez on 19/03/2019.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+/*
+* Copyright (C) 2018, ownCloud GmbH.
+*
+* This code is covered by the GNU Public License Version 3.
+*
```

Comment:
License Header is missing

Reply:
That it is the license header. Maybe do you mean in other file?